elastic-beats/templates/topbeat.yml.j2

```diff
@@ -1,165 +0,0 @@
-################### Topbeat Configuration Example #########################
-
-############################# Input ############################################
-input:
-  period: 10
-  procs: ["php.*", "nginx.*","java.*","docker.*","mongo.*","salt.*","python.*","collectdmon.*"]
```

Comment:
this config needs to be transferred to metricbeats.  this reduces the disk space requirement and is targeted for our applications. this should probably be set based on the elastic-beats:metricbeat pillar